### ISSUE TITLE
`UnusedKeywordOperation` check should ignore `Keyword.validate!/2`

### ISSUE THREAD

**Issue by cgrothaus** [rrrene/credo#1128]
### Environment

* Credo version: 1.7.5-ref.main.a6568e1ca
* Erlang/Elixir version: Erlang/OTP 26 [erts-14.2.2] [source] [64-bit] [smp:8:8] [ds:8:8:10] [async-threads:1] [jit]; Elixir 1.16.1 (compiled with Erlang/OTP 26)
* Operating system: mac OS

### What were you trying to do?
Use [Keyword.validate!/2](https://hexdocs.pm/elixir/1.16.1/Keyword.html#validate!/2), like in the following example:
```elixir
defmodule Example do
  def function(arg1, opts \\ []) do
    # credo warns about the following line:
    Keyword.validate!(opts, [:base_url, :meta, :receive_timeout])

    # just some random code:
    base_url = Keyword.get(opts, :base_url) || ""
    arg1 <> base_url
  end
end
```

### Expected outcome
No Credo warnings.

### Actual outcome
Credo warns about _unused return values for Keyword functions_:
```
Checking 1324 source files (this might take a while) ...

  Warnings - please take a look                                                                                                                                                                                                                                    
┃ 
┃ [W] ↗ There should be no unused return values for Keyword functions.
┃       lib/example.ex:4:5 #(Example.function)
```

But for `Keyword.validate!/2`, we really do not need to use the return value.

**Comment by rrrene** [comment]
Thx for reporting! :+1:

**Comment by rrrene** [comment]
Thanks for reporting this 😀  It is now fixed on `master`.

You can try this by setting the Credo dep to

```elixir
{:credo, github: "rrrene/credo"}
```

Please report back if your issue is solved! 👍

**Comment by cgrothaus** [comment]
Solved 🥳 ! Thanks for reacting so insanely fast 🏎️ .

**Comment by rrrene** [comment]
This is live in `v1.7.6` 🎉
